### DISCUSSION TITLE
#pypulseq on Slack

### DISCUSSION THREAD

**Discussion by sravan953** [imr-framework/pypulseq#97] [General]
Everyone is welcome to join the PyPulseq Slack channel. Please:
1. Join the IARG workspace -- https://join.slack.com/t/iarg-cmrrc/shared_invite/zt-1t3s2u8lz-1EJt3ufWxYvzEab0cZyfdA
2. Join the #pypulseq channel

(Slack links expire after a while, so please reply to this thread for an updated link and I can then edit this post with the new links)

**Comment by congzhangzh** [comment]
Hi Keerthi,

It’s expired, can you update it? @sravan953 

Best regards,
Cong

**Reply by sravan953** [reply]
Done!

**Comment by congzhangzh** [comment]
Cool:)
